build-linux/build.sh

```diff
@@ -8,6 +8,7 @@ source "${THISDIR}"/../helpers.sh
 
 ARCH="$1"
 TOOLCHAIN="$2"
+export KBUILD_OUTPUT="$3"
```

Comment:
no strong opinion here, but instead of being a required parameter, could it have been an optional env var.

```
export KBUILD_OUTPUT=${KBUILD_OUTPUT:-"."}
```

Reply:
I really want to do away with global variables being accessed at random locations and rather make them explicit arguments instead, to be honest.